src/pages/p2p/components/_dp2p.js

```diff
@@ -91,6 +101,21 @@ const Row = styled.div`
         margin-top: 40px;
     }
 `
+
+const StyledIFrame = styled.iframe`
+    height: 315px;
+    width: 100%;
+    max-width: 560px;
+
+    @media ${device.tablet} {
+        width: 90%;
+    }
+
+    @media ${device.mobileM} {
+        width: 90%;
+    }
```

Comment:
```suggestion
```

Reply:
Actually in fact I think we can just remove this entirely, just noticed that your container had already paddings included. Last one I promise 🙏